Add merge for ininstance calls to improve readbility

HarshCasper opened the pull request:
## Description 

This Pull Request does the following: 

- Add merge for `ininstance` calls to improve readability

The Issue seems to be very minor but it improves the readability of the Code for the better.

I'm a First-Time Contributor to Zulip Open-Source so it would be helpful for me understanding how PR workflow works and help me step up in future contributions as well.

ligmitz reviewed (commented):
Hi there, I would like to tell you that there is a certain format of writing git commit message in zulip. You can have a look at the [docs](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages). It is a compulsion to adhere to it. Thanks

HarshCasper commented:
Hi @ligmitz

Thanks for reviewing the PR. I have updated the Commit message as you suggested and removed the typo error as well. I would be glad if you can review the PR now.

ligmitz reviewed (commented):
Looks good to me ! However you don't need to close and open a PR again for force pushing your commits.

HarshCasper commented:
Hi @ligmitz 

Thanks for the review. I tried to do a lazy way of soft resetting the commits force pushing a single commit which closed the PR automatically. 

Can you please let me know any steps I have to follow for getting my PR merged now?

ligmitz commented:
You can wait until someone with write access take a look at your PR or you can bring this up in the [zulip community chat](https://chat.zulip.org/). Please be patient until then.

andersk commented:
There’s still a typo in the commit message and the PR title: “`ininstance`” should be “`isinstance`”. Additionally, commit messages [should](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages) be written in the imperative (“Merge …”, not “Adding merge for …”), and end with a period.

timabbott commented:
@HarshCasper have you had a chance to fix your commit message?

HarshCasper commented:
Hello @timabbott 

This Message slipped from my Dashboard. Apologies for that. I will fix this in a few hours and update here.
